### DISCUSSION TITLE
Looking for contributors on the unofficial VSCode extension

### DISCUSSION THREAD

**Discussion by pomdtr** [excalidraw/excalidraw#4948] [Show and tell]
Hi ! 
I'm the author of the unofficial [Excalidraw Schema Editor](https://marketplace.visualstudio.com/items?itemName=pomdtr.excalidraw-editor) VSCode extension.

I'm currently struggling on two issues (unrelated to excalidraw):

- https://github.com/pomdtr/vscode-excalidraw-embed/issues/10
- https://github.com/pomdtr/vscode-excalidraw-embed/issues/12

The second one could be particulary interesting to the community since it would allow to edit excalidraw files directly from github.

I would love to get some help from the community on these !

**Comment by dwelle** [comment]
Hey @pomdtr — thanks for working on the extension! Once we're free we'll gladly help, but right now all of us are super busy. That said, if you have any specific questions regarding the API or similar, feel free to ask here or on the Discord channel [discord.gg/UexuTaE](https://t.co/wClmtW7mbA) (you can also PM me directly there).

**Reply by pomdtr** [reply]
To be honest I struggle a lot more with vscode API than on excalidraw. Thanks for releasing such an awesome component, it was a piece of cake to integrate !
